### ISSUE TITLE
Modify Subtitle Appearance and Timing - Feature Request

### ISSUE THREAD

**Issue by userkace** [huynhsontung/Screenbox#143]
Ability to change the appearance of subtitle (even just sizing) and timing so that we can sync it better.

**Comment by huynhsontung** [comment]
Subtitle time sync is planned to be implemented soon. As for customizing the appearance, due to LibVLC limitations, it is not possible to provide a good user experience to change these settings. This task requires a substantial amount of effort to improve VLC's subtitle rendering. I will update here when there is progress on this topic.

**Comment by userkace** [comment]
The font itself is great; its just that sizes on different display vary on the display scale, causing it to block substantial content on other sizes. Just a dynamic size or ability to pick between preset size could help with user experience.

**Comment by huynhsontung** [comment]
Not a perfect solution but you can change the subtitle font size by using the "Advanced mode" in the settings. Add `--freetype-rel-fontsize=10` into "Global arguments", where 10 is the font size value that you can customize. Please note that **bigger** values mean **smaller** text.

**Comment by Venderbad** [comment]
What about the colors? Sometimes the default subtitle color (is it pure black?) makes the text almost invisible in videos with dark backgrounds. I think it would be nice to have a toggler to override the color setting of the subtitle text.

Nice app by the way. Shall I send this in #29 ?

**Comment by cbordeman** [comment]
Due to very poor eyesight,  I need to also have a black, solid background (or a thick outline).  Is this possible?

**Comment by huynhsontung** [comment]
@cbordeman Yes you can, also using "Advanced mode" and add this argument to the global arguments: `--freetype-background-opacity=255`

Ref: https://wiki.videolan.org/VLC_command-line_help/
>       --freetype-background-opacity=<integer [0 .. 255]> 
>                                 Background opacity

**Comment by cbordeman** [comment]
@huynhsontung Thanks, works perfectly.
